en/components/date_picker.md

```diff
@@ -125,6 +125,42 @@ Here is the formatted date:
 </div>
 <div class="divider--half"></div>
 
+#### Templating Input Group
+
+The Date Picker's input group look can be customized. To do that we need to decorate a ng-template inside the date picker with `IgxDatePickerTemplate` directive and use the context returned to customize the way the date picker's input is displayed. You can make use of `value`, `displayData` and the `openDialog`.
```

Comment:
See my comment for the time picker. Copy the modified texts into the JP versions as well.